src/components/Dashboard.jsx

```diff
@@ -51,8 +51,8 @@ const Dashboard = (pageData) => {
     <div className="govuk-width-container">
       <div className="govuk-grid-row">
         <div className="govuk-grid-column-one-half">
-          <h2 className="govuk-heading-m">Create a new notification</h2>
-          <p className="govuk-body">You can use the online form to create and submit an Advance Voyage Notification to UK Border Force.</p>
+          <h2 className="govuk-heading-m">Create a new report</h2>
+          <p className="govuk-body">You can use this online form to create and submit a pleasure craft report to UK Border Force, notifying them of your travel plans.</p>
```

Comment:
Shall we be consistent and use `Pleasure Craft Report` in the same form as [the legislation](https://www.legislation.gov.uk/uksi/1996/1406/made) and the rest of the places in the app?

```suggestion
          <p className="govuk-body">You can use this online form to create and submit a Pleasure Craft Report to UK Border Force, notifying them of your travel plans.</p>
```